### DISCUSSION TITLE
EcoFlow Integration not working

### DISCUSSION THREAD

**Discussion by bluefire100** [community/community#44761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Cannot install Ecoflow Integration into HACS because of this error:
**_**_No manifest.json file found 'custom_components/ecoflow_iot/manifest.json'_**_**

How to fix?

**Comment by mykhailog** [comment]
It looks like you post in wrong channel. But please try again. I've just updated the integration.
